src/web/app/pages-instructor/instructor-session-individual-extension-page/individual-extension-date-modal/individual-extension-date-modal.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="modal-header alert-primary">
+  <h5 class="modal-title">Select the new deadline for extension</h5>
+  <button type="button" class="close" (click)="activeModal.dismiss()">
+    <i class="fas fa-times"></i>
+  </button>
+</div>
+<div class="modal-body">
+  Extending the feedback session for <b>{{numStudents}}</b> student(s) and <b>{{numInstructors}}</b> instructor(s):
```

Comment:
nit: To make it consistent with the other variables in this file
```suggestion
  Extending the feedback session for <b>{{ numStudents }}</b> student(s) and <b>{{ numInstructors }}</b> instructor(s):
```